src/fa/game_session.py

```diff
@@ -177,13 +183,46 @@ def _turn_state_changed(self, val):
 
     def _launched(self):
         self._logger.info("Game has started")
+        self._add_to_full_gamelog()
+
+    def _limit_replayid_gamelogs(self):
+        nb_replayid_files = 0
+        files = os.listdir(util.LOG_DIR)
+        files = map(lambda f: os.path.join(util.LOG_DIR, f), files)
+        files = sorted(files, key = os.path.getmtime)
```

Comment:
Either I was wrong here or didn't catch this one - no whitespace around keyword parameter assignment